scripts/deploy.sh

```
@@ -11,19 +11,19 @@ if [ -z "$ENV" ] || [[ ! "$ENV" =~ ^(dev|prod|all)$ ]]; then
 fi
 
 cd "$PROJECT_ROOT"
+source "$PROJECT_ROOT/scripts/lib-deploy.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./scripts/lib-deploy.sh was not specified as input (see shellcheck -x).

(SC1091)